corehq/apps/accounting/static/accounting/js/confirm_plan.js

```diff
@@ -17,9 +17,35 @@ hqDefine('accounting/js/confirm_plan', [
         self.currentPlan = currentPlan;
         self.newPlan = newPlan;
 
-        // If the user is upgrading, don't let them continue until they have agreed to the therms
+        // If the user is upgrading, don't let them continue until they agree to the minimum subscription terms
         self.userAgreementSigned = ko.observable(!isUpgrade);
 
+
+
+        self.downgradeReasonList = ko.observableArray([
```

Comment:
I think these lists of reasons can be regular arrays - they'd need to be observables if you were adding/removing objects and needed to watch those changes.